Heejung/Lv2/H-Index.js

```diff
@@ -0,0 +1,8 @@
+function solution(citations) {
+  for (let h = citations.length; h >= 0; h -= 1) {
+    // h번 이상 인용된 논문 개수
+    const cnt = citations.filter((citation) => citation >= h).length;

```

Comment:
```suggestion
    const cnt = citations.reduce((acc, cur) => acc += (cur >= h), 0);
```
배열을 새로 생성하는게 메모리를 많이 잡아먹다보니 filter보다는 reduce를 쓰면 메모리를 덜 쓸거 같아요.